### PR TITLE
LIS Area: consolidate into LIS_DEPLOY

### PR DESCRIPTION
**Pull request by chvalean**
1. Make obsolete the LIS tag, just move them to CORE area to avoid too much disruption in pipelines
2. Rename tests to remove the LIS from title, tests are generic.
3. Move LIS RPMs related tests to the LIS_DEPLOY xml and area.

### CODE REVIEW

**Comment by chvalean** [comment]
@iamshital @johnsongeorge-w please approve this but do not merge, will do that tomorrow after I also check some of the pipelines we maintain to remove the LIS area wherever needed.
Thank you!

**Review by johnsongeorge-w — APPROVED** [review]

File: XML/TestCases/LIS-Tests.xml [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+<TestCases>
     <test>
         <testName>LIS-DRIVER-VERSION-CHECK</testName>
         <testScript>LIS-VERSION-CHECK.sh</testScript>
         <files>.\TestScripts\Linux\LIS-VERSION-CHECK.sh,.\TestScripts\Linux\utils.sh</files>
         <setupType>OneVM</setupType>
         <Platform>Azure</Platform>
         <Category>Functional</Category>
-        <Area>LIS</Area>
+        <Area>LIS_DEPLOY</Area>
```
Comment (juhlee-microsoft) [inline XML/TestCases/LIS-Tests.xml:9]: @chvalean this introduces new AREA. Any reason beyond 'LIS'?

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

File: XML/TestCases/LIS-Tests.xml (hunk 1 above)
Comment (chvalean) [reply XML/TestCases/LIS-Tests.xml:9]: > @chvalean this introduces new AREA. Any reason beyond 'LIS'?

LIS_DEPLOY already exists for the rpm install scenarios.
We actually get rid of one area - the one called LIS.

Comment (juhlee-microsoft) [reply XML/TestCases/LIS-Tests.xml:9]: acked. From now on, there should be no 'LIS' in Area. Can you create an issue for 13 'LIS' Area in repo, @chvalean ?

**Review by juhlee-microsoft — APPROVED** [review]

**Comment by juhlee-microsoft** [comment]
> @iamshital @johnsongeorge-w please approve this but do not merge, will do that tomorrow after I also check some of the pipelines we maintain to remove the LIS area wherever needed.
> Thank you!

Missed this comment and merged already. Please next time use the built-in label.

**Comment by chvalean** [comment]
> > @iamshital @johnsongeorge-w please approve this but do not merge, will do that tomorrow after I also check some of the pipelines we maintain to remove the LIS area wherever needed.
> > Thank you!
> 
> Missed this comment and merged already. Please next time use the built-in label.

Will do. This one was fine, already had an email to Jonson and Shital, we were ready for this merge.
Thank you!

**Pull request merged**